lifecycle/src/test/scala/it/pagopa/interop/agreementprocess/lifecycle/AttributesRulesSpec.scala

```diff
@@ -476,5 +477,52 @@ class AttributesRulesSpec extends AnyWordSpecLike {
 
       verifiedAttributesSatisfied(agreement, eService, consumer) shouldBe false
     }
+
+    "return false if at least one single attribute with REVOKE_ON_EXPIRATION is expired" in {
+      val producerId = UUID.randomUUID()
+      val attr1      = UUID.randomUUID()
+      val attr2      = UUID.randomUUID()
+
+      val eServiceAttr = SpecData
+        .catalogVerifiedAttribute()
+        .copy(verified = Seq(SpecData.catalogSingleAttribute(attr1), SpecData.catalogSingleAttribute(attr2)))
+
+      val tenantAttr = Seq(
+        SpecData.tenantVerifiedAttribute(attr1, producerId),
+        SpecData.tenantVerifiedAttribute(id = attr2, verifierId = producerId, renewal = REVOKE_ON_EXPIRATION)
```

Comment:
it's a bit difficult to understand the test case if the test values are taken as default parameter of the helper method.
I suggest to explicitly set them like

```suggestion
        SpecData.tenantVerifiedAttribute(id = attr2, verifierId = producerId, renewal = REVOKE_ON_EXPIRATION, extensionDate = OffsetDateTime.now().minusDays(3).some)
```

Same for the other test